app/src/components/staking/PageValidators.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <TmPage :managed="true" hide-header>
+  <TmPage>
     <div
```

Comment:
use TmPage loading

Reply:
ah so this is a bit tricky because we have the svg loader. should we / can we pass this in to TmPage?

Reply:
yes. you can define the svg as the loader component